tutorials/image/cifar10_estimator/cifar10_model.py

```diff
@@ -17,9 +17,8 @@
 from __future__ import division
 from __future__ import print_function
 
-import tensorflow as tf
-
 import model_base
+import tensorflow as tf
```

Comment:
This was correct the way it was before. You want to import libraries before you import your own files.